tests/settings.py

```diff
@@ -96,3 +96,9 @@
 }
 
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
+
+# Suppresses warning for versions of Wagtail above 3.0
+if WAGTAIL_VERSION >= (3, 0):
```

Comment:
As mentioned earlier I don't think this conditional adds any value - `WAGTAILADMIN_BASE_URL` is a valid setting on any version, even if it only raises a warning on >=3.0. No worries - I'll roll this back when I merge.